### DISCUSSION TITLE
Support Multi-threadding

### DISCUSSION THREAD

**Discussion by vijayaudupa** [armenak/DataDefender#195] [Ideas]
When trying to anonymize large data sets in database tables, example, a table with 20,000,000 rows in a table, TAT is high (approx. 24+ hours). Would it be possible to bring down TAT if the process can be complete using multiple threads? The thread pool size can be an external property to control parallelism and default can be current behavior of single threads.

**Comment by armenak** [comment]
Hello and thank you for the request. It is definitely possible, but I will need some time to implement it. I will keep this ticket open and will update it when it is done. At this point, I do not have an ETA.
